### PR TITLE
Vector of shared ptrs Homework

### PR DESCRIPTION
**Pull request by runer53248**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: vector-of-shared-ptrs
🏅 Granted 5 XP for @runer53248

File: homework/vector-of-shared-ptrs/test.cpp [hunk 1]
```diff
@@ -71,6 +71,17 @@ TEST_F(Homework, ShouldSubtract10ForVector) {
...
+    std::vector<std::shared_ptr<int>> vec(count);
+    for (auto& num : vec) {
+      num = nullptr;
+    }
+    sub10(vec);
+    for (size_t i = 0 ; i < count ; ++i) {
+        EXPECT_EQ(vec[i], nullptr);
+    }
+}
+
```
Comment (nauka-programowania-MA) [inline homework/vector-of-shared-ptrs/test.cpp:84]: Nice! :)

File: homework/vector-of-shared-ptrs/vectorFunctions.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#include "vectorFunctions.hpp"
+#include <algorithm>
+#include <iostream>
+
+myVector generate(int count) {
+    myVector result(count);
+    std::for_each(begin(result), end(result),
+                  [number = 0](auto& pointer) mutable {
+                      pointer = std::make_shared<int>(number++);
+                  });
```
Comment (nauka-programowania-MA) [inline homework/vector-of-shared-ptrs/vectorFunctions.cpp:10]: This is a little to complicated for such simple thing :) Better use normal loop :) or std::generate

File: homework/vector-of-shared-ptrs/vectorFunctions.cpp [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    return result;
+}
+
+void print(const myVector& vectorOfPointers) {
+    for (const auto& pointer : vectorOfPointers) {
+        if(pointer){
```
Comment (nauka-programowania-MA) [inline homework/vector-of-shared-ptrs/vectorFunctions.cpp:16]: space

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**